src/components/select-field/BaseSelectField.js

```diff
@@ -469,18 +546,22 @@ class BaseSelectField extends React.Component<Props, State> {
             >
                 <PopperComponent placement={dropdownPlacement} isOpen={isOpen} modifiers={dropdownModifiers}>
                     {this.renderSelectButton()}
-                    <SelectFieldDropdown
-                        isScrollable={isScrollable}
-                        multiple={multiple}
-                        selectedValues={selectedValues}
-                        selectFieldID={this.selectFieldID}
-                    >
-                        {this.renderSelectOptions()}
-                    </SelectFieldDropdown>
+                    <>
```

Comment:
i dont think you need this fragment

Reply:
Yeah this should be removed. It was originally added since I had the search field as a sibling to SelectorDropdown